cognite/neat/_graph/transformers/_classic_cdf.py

```diff
@@ -295,7 +302,22 @@ def __init__(
     ?entity classic:externalId "{external_id}" .
 }}"""
 
+    @staticmethod
+    def create_lookup_entity(graph: Graph, namespace: Namespace) -> Callable[[str, str], URIRef]:
+        @lru_cache(maxsize=10_000)
+        def lookup_entity(entity_type: str, external_id: str) -> URIRef:
+            query = RelationshipAsEdgeTransformer._lookup_entity_query.format(
```

Comment:
we should add classic namespace to [default_prefixes](https://github.com/cognitedata/neat/blob/5afc4a2444b6a5106abd63f91aad8b6c372e9bb5/cognite/neat/_constants.py#L74) and make sure they are all registered when [upserting prefixes in store](https://github.com/cognitedata/neat/blob/5afc4a2444b6a5106abd63f91aad8b6c372e9bb5/cognite/neat/_store/_base.py#L99)